### PR TITLE
Reverts Mesons not working on lavaland

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
add: Mesons work on lavaland again.
/:cl:

Why: Our lighting system still just doesn't update fast enough at higher populations so you're constantly having to stop in the dark. The secret ruins have not materialized.

### CODE REVIEW

File: code/modules/clothing/glasses/glasses.dm [hunk 1]
```diff
@@ -39,46 +39,6 @@
...
 	glass_colour_type = /datum/client_colour/glass_colour/lightgreen
-	var/static/list/meson_mining_failure_excuses = list("seismic activity", "excessive lava", "ambient radiation", "electromagnetic storms", "bluespace disruption", \
-	"gravity", "dust", "dense rock", "ash", "badly understood science", "radiant heat")
-	var/picked_excuse
-	var/mesons_on = TRUE
-
-/obj/item/clothing/glasses/meson/Initialize()
-	. = ..()
-	picked_excuse = pick(meson_mining_failure_excuses)
-	START_PROCESSING(SSobj, src)
```
Comment (ChangelingRain) [inline code/modules/clothing/glasses/glasses.dm]: you do actually need this

File: code/modules/clothing/glasses/glasses.dm [hunk 2]
```diff
@@ -39,46 +39,6 @@
...
-
-/obj/item/clothing/glasses/meson/Destroy()
-	STOP_PROCESSING(SSobj, src)
```
Comment (ChangelingRain) [inline code/modules/clothing/glasses/glasses.dm]: and this

**Comment by Kromgar** [comment]
>When you run into a chasm because lighting doesn't update

**Comment by DoctoreMad** [comment]
Please stop the memories hurt

**Comment by SgtxRootbeer** [comment]
fix this, and speedmerge it please...
i want to enjoy mining again...

**Comment by FantasticFwoosh** [comment]
Remove the advanced lighting then, because we can't keep going in circles for a costly system if its going to roadblock whatever we do on technical limits beyond what we can currently cope with.

> We've had multiple PR's exclusively just to fix problems associated with it, and light updating is a generally awful experience.

@optimumtact isn't exactly going to stop you.

**Comment by Repukan** [comment]
Fwoosh he's only reverting it because the major selling point of the meson change was "secret ruins" and they haven't happened after a good amount of time.

Lighting system is sadly here to stay

**Comment by Cyberboss** [comment]
@KorPhaeron

File: code/modules/clothing/glasses/glasses.dm (hunk 1 above)
Comment (KorPhaeron) [inline code/modules/clothing/glasses/glasses.dm]: Why for? It looks like the only things meson do in process is turn on/off

Comment (ChangelingRain) [inline code/modules/clothing/glasses/glasses.dm]: Subtypes.

Comment (KorPhaeron) [inline code/modules/clothing/glasses/glasses.dm]: Shouldn't the subtypes be the ones adding themselves to processing then

**Comment by Repukan** [comment]
Don't you die on me baby

**Comment by Kromgar** [comment]
@KorPhaeron please save us

**Comment by kevinz000** [comment]
kor save us

**Comment by SgtxRootbeer** [comment]
KOR PLEASE, I BEG OF YOU.

**Comment by KorPhaeron** [comment]
soon(TM)

**Comment by Cyberboss** [comment]
Stale

```
4:11:06 PM
%Cyberboss Heads I close meson revert 
4:11:08 PM %Cyberboss !coin 
4:11:09 PM Bot32 Cyberboss: Heads! 
```

**Comment by Bawhoppen** [comment]
>inb4 mass REEEEs

**Pull request closed without merging**